Src/Common.props

```diff
@@ -25,6 +25,13 @@
     <PackageIconUrl>https://raw.githubusercontent.com/AutoFixture/AutoFixture/79c882c3f4af3cf52ad43e5c95851f25d217ac17/AutoFixtureLogo200x200.png</PackageIconUrl>
     <PackageIcon>icon.png</PackageIcon>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+
+    <!--Prevents restoring the latest version of FSharp.Core (i.e. FSharp.Core 5.0.0)-->
+    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
```

Comment:
I'm a bit worried that we add these properties for all the projects, not F# related only. These properties are very "magic" and it's a bit hard to remember specifics for this particular repo.

Are they strictly necessary for all the projects?

Reply:
@zvirja I agree, it would make sense to create a props file specific for F# projects, it would make the fix more explicit. I will make the necessary adjustments.